scripts/git/release.sh

```diff
@@ -0,0 +1,138 @@
+
+#!/usr/bin/env bash -e
```

Comment:
1. this should be on the first line
2. Windows Bash doesn support `bash -e` 
3. also add copyright on line 2

lines 1-4:
```
#!/usr/bin/env bash
# Copyright (c) Microsoft. All rights reserved.
# Note: Windows Bash doesn't support shebang extra params
set -e
```